nameko/amqp.py

```diff
@@ -38,12 +39,20 @@ class TestTransport(Transport):
     Connection = ConnectionTester
 
 
-def verify_amqp_uri(amqp_uri):
-    connection = Connection(amqp_uri)
+def verify_amqp_uri(amqp_uri, config=None):
+    ssl_params = get_ssl_params(config if config else {})
+    connection = Connection(amqp_uri, ssl=ssl_params)
     if connection.transport_cls != 'amqp':
         # Can't use these heuristics. Fall back to the existing error behaviour
         return
 
     transport = TestTransport(connection.transport.client)
     with transport.establish_connection():
         pass
+
+
+def get_ssl_params(config):
+    params = config.get('SSL')
+    if params and isinstance(params, dict):
+        params.update({'cert_reqs': ssl.CERT_REQUIRED})
```

Comment:
maybe we should at least look for this key as well to make it possible for users to set the behaviour if they want? (e.g. params['cert_reqs'] = params.get('cert_reqs', CERT_REQUIRED)` or similar

or is this never the case?